### PR TITLE
Add isRedHat test

### PR DESCRIPTION
**Pull request by sebrandon1**
Adds the `isRedHatRelease` test to the new test suite.

### CODE REVIEW

File: cnf-certification-test/platform/suite.go [hunk 1]
```diff
@@ -173,3 +179,28 @@ func testTainted(env *provider.TestEnvironment) {
...
+		ginkgo.By(fmt.Sprintf("%s(%s) is checked for Red Hat version", cut.Podname, cut.Data.Name))
+		baseImageTester := isredhat.NewBaseImageTester(common.DefaultTimeout, clientsholder.NewClientsHolder(), clientsholder.Context{
+			Namespace:     cut.Namespace,
+			Podname:       cut.Podname,
+			Containername: cut.Data.Name,
+		})
+
+		result, err := baseImageTester.TestContainerIsRedHatRelease()
+		if err != nil {
+			logrus.Debug("failed to collect release information from container: ")
```
Comment (hamadise) [inline cnf-certification-test/platform/suite.go:197]: logrus.Error

File: cnf-certification-test/platform/suite.go [hunk 2]
```diff
@@ -173,3 +179,28 @@ func testTainted(env *provider.TestEnvironment) {
...
+		}
+		if !result {
+			failedContainers = append(failedContainers, cut.Namespace+"/"+cut.Podname+"/"+cut.Data.Name)
+			tnf.ClaimFilePrintf("Container: %s/%s (ns: %s) has failed the RHEL release check", cut.Podname, cut.Data.Name, cut.Namespace)
```
Comment (edcdavid) [inline cnf-certification-test/platform/suite.go:201]: use StringShort() here and in other places to display the container

File: cnf-certification-test/platform/suite.go [hunk 3]
```diff
@@ -173,3 +179,28 @@ func testTainted(env *provider.TestEnvironment) {
 	gomega.Expect(taintedNodes).To(gomega.BeNil())
 	gomega.Expect(errNodes).To(gomega.BeNil())
 }
+
+func testIsRedHatRelease(env *provider.TestEnvironment) {
+	ginkgo.By("should report a proper Red Hat version")
+	failedContainers := []string{}
+	logrus.Infof("looping through %d containers", len(env.Containers))
```
Comment (edcdavid) [inline cnf-certification-test/platform/suite.go:186]: logrus.debugf

Comment (sebrandon1) [reply cnf-certification-test/platform/suite.go:186]: Whoops, this was just a debugging line.  I'll remove in my next commit.

File: cnf-certification-test/platform/suite.go (hunk 1 above)
Comment (sebrandon1) [reply cnf-certification-test/platform/suite.go:197]: Done.

File: cnf-certification-test/platform/suite.go (hunk 2 above)
Comment (sebrandon1) [reply cnf-certification-test/platform/suite.go:201]: Done.

**Review by edcdavid — APPROVED** [review]

File: cnf-certification-test/platform/suite.go [hunk 4]
```diff
@@ -173,3 +179,27 @@ func testTainted(env *provider.TestEnvironment) {
 	gomega.Expect(taintedNodes).To(gomega.BeNil())
 	gomega.Expect(errNodes).To(gomega.BeNil())
 }
+
+func testIsRedHatRelease(env *provider.TestEnvironment) {
+	ginkgo.By("should report a proper Red Hat version")
+	failedContainers := []string{}
+	for _, cut := range env.Containers {
+		ginkgo.By(fmt.Sprintf("%s(%s) is checked for Red Hat version", cut.Podname, cut.Data.Name))
```
Comment (greyerof) [inline cnf-certification-test/platform/suite.go:187]: Please, add the namespace to this "By" statement (or use the cut.StringShort()). Also separate podname and cut name formatting chars with an extra space char: "%s (%s)"

File: cnf-certification-test/platform/suite.go [hunk 5]
```diff
@@ -173,3 +179,27 @@ func testTainted(env *provider.TestEnvironment) {
...
+		baseImageTester := isredhat.NewBaseImageTester(common.DefaultTimeout, clientsholder.NewClientsHolder(), clientsholder.Context{
+			Namespace:     cut.Namespace,
+			Podname:       cut.Podname,
+			Containername: cut.Data.Name,
+		})
+
+		result, err := baseImageTester.TestContainerIsRedHatRelease()
+		if err != nil {
+			logrus.Error("failed to collect release information from container: ")
```
Comment (greyerof) [inline cnf-certification-test/platform/suite.go:196]: Add the "err" variable value to this error trace, please.

File: cnf-certification-test/platform/suite.go (hunk 4 above)
Comment (sebrandon1) [reply cnf-certification-test/platform/suite.go]: Added the `StringShort()` recommendation.  👍

**Pull request merged**